### PR TITLE
remove stacks roadmap, update links

### PR DESCRIPTION
**Pull request by ChuckHend**

### CODE REVIEW

File: docusaurus/docs/tembo-stacks/intro-to-stacks.md [hunk 1]
```diff
@@ -6,23 +6,21 @@ sidebar_position: 1
...
 |[OLTP](oltp.md)| Amazon RDS |
 |[OLAP](olap.md)| Snowflake, Bigquery |
-|[Machine Learning](enterprise-llm.md)| Pinecone, ChromaDB |
+|[Machine Learning](machine-learning.md)| MindsDB |
 |[Message Queue](message-queue.md)| Amazon SQS, RabbitMQ, Redis |
-|[Data Warehouse](olap.md)| Snowflake, Bigquery |
+|[Data Warehouse](datawarehouse.md)| Snowflake, Bigquery |
 |Mongo Alternative on Postgres| MongoDB | 
 |Geospatial|  | 
 |[Vector DB](vector-db.md)| Pinecone, Weaviate |
```
Comment (samay-sharma) [inline docusaurus/docs/tembo-stacks/intro-to-stacks.md:26]: Can you add a line at the end of this table saying something like "We're working on additional stacks and the roadmap for that is here (roadmap.tembo.io): Please upvote stacks you'd like" etc.

File: docusaurus/docs/tembo-stacks/intro-to-stacks.md [hunk 2]
```diff
@@ -6,23 +6,21 @@ sidebar_position: 1
...
 |----|---------------|
 |[OLTP](oltp.md)| Amazon RDS |
 |[OLAP](olap.md)| Snowflake, Bigquery |
-|[Machine Learning](enterprise-llm.md)| Pinecone, ChromaDB |
+|[Machine Learning](machine-learning.md)| MindsDB |
 |[Message Queue](message-queue.md)| Amazon SQS, RabbitMQ, Redis |
-|[Data Warehouse](olap.md)| Snowflake, Bigquery |
+|[Data Warehouse](datawarehouse.md)| Snowflake, Bigquery |
 |Mongo Alternative on Postgres| MongoDB | 
 |Geospatial|  |
```
Comment (samay-sharma) [inline docusaurus/docs/tembo-stacks/intro-to-stacks.md:25]: Let's call out ESRI here.

Comment (samay-sharma) [reply docusaurus/docs/tembo-stacks/intro-to-stacks.md:25]: Maybe even Oracle Spatial

File: docusaurus/docs/tembo-stacks/intro-to-stacks.md (hunk 1 above)
Comment (ChuckHend) [reply docusaurus/docs/tembo-stacks/intro-to-stacks.md:26]: yes!

File: docusaurus/docs/tembo-stacks/intro-to-stacks.md [hunk 3]
```diff
@@ -6,28 +6,28 @@ sidebar_position: 1
...
 |[OLTP](oltp.md)| Amazon RDS |
 |[OLAP](olap.md)| Snowflake, Bigquery |
-|[Machine Learning](enterprise-llm.md)| Pinecone, ChromaDB |
+|[Machine Learning](machine-learning.md)| MindsDB |
 |[Message Queue](message-queue.md)| Amazon SQS, RabbitMQ, Redis |
-|[Data Warehouse](olap.md)| Snowflake, Bigquery |
+|[Data Warehouse](datawarehouse.md)| Snowflake, Bigquery |
 |Mongo Alternative on Postgres| MongoDB | 
-|Geospatial|  | 
+|Geospatial| Estri, Oracle |
```
Comment (samay-sharma) [inline docusaurus/docs/tembo-stacks/intro-to-stacks.md:25]: Typo. Also, it's capitalized ESRI

File: docusaurus/docs/tembo-stacks/intro-to-stacks.md [hunk 4]
```diff
@@ -6,28 +6,28 @@ sidebar_position: 1
...
 |[Message Queue](message-queue.md)| Amazon SQS, RabbitMQ, Redis |
-|[Data Warehouse](olap.md)| Snowflake, Bigquery |
+|[Data Warehouse](datawarehouse.md)| Snowflake, Bigquery |
 |Mongo Alternative on Postgres| MongoDB | 
-|Geospatial|  | 
+|Geospatial| Estri, Oracle | 
 |[Vector DB](vector-db.md)| Pinecone, Weaviate |
 |[Standard](standard.md)| Amazon RDS |
 
+We are actively working on additional Stacks. Check out the [Tembo Roadmap](https://roadmap.tembo.io/roadmap) and upvote your favorites and Stack you'd like to see next.
```
Comment (samay-sharma) [inline docusaurus/docs/tembo-stacks/intro-to-stacks.md:29]: Check out the Tembo Roadmap and upvote the stacks you'd like to see next.

**Review by samay-sharma — APPROVED** [review]

**Pull request merged**